mrp_bom_hierarchy/__manifest__.py

```diff
@@ -6,12 +6,13 @@
     "summary": "Make it easy to navigate through BoM hierarchy.",
     "version": "16.0.1.0.1",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "contributors": "Grap",
```

Comment:
does this key even exist?

Reply:
Yes, here https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/models/ir_module.py#L729

Reply:
interesting, TIL, thanks!